test/webdriver/test-basic.js

```diff
@@ -14,18 +14,42 @@ describe('application launch', function() {
   this.timeout(30000);
   const executable = getElectronPath();
 
-  beforeEach(function() {
+  beforeEach(async function() {
     this.app = new Application({
       path: executable,
       args: [getRoot()],
       env: process.env,
     });
-    return this.app.start();
+    await this.app.start();
+    expect(this.app.isRunning()).toBe(true);
+    await this.app.client.waitUntilWindowLoaded();
+
+    // Wait for both BrowserWindow and first tab webview to exist
+    await this.app.client.waitUntil(function() {
+      return this.getWindowCount().then((count) => count === 2);
+    });
   });
 
-  it('shows an initial window', async function() {
+  it('shows an initial window and tab', async function() {
+    let { value: url } = await this.app.client.url();
+    expect(url.startsWith('file://')).toBe(true);
+    expect(url.endsWith('browser.html')).toBe(true);
+
     const count = await this.app.client.getWindowCount();
```

Comment:
Worth a comment explaining this?  You and I know, and there's clues above -- but this is *very* surprising, or at least it was to me.